Source/SIMPLib/CoreFilters/CreateDataArray.cpp

```diff
@@ -110,27 +117,58 @@ std::vector<T> createInitVector(const QString& initializationValue, int32_t numC
     {
       for(int32_t i = 0; i < numComponents; i++)
       {
-        values[i] = convert.convert(tokens[0]);
+        conversionResult = convert.convert(tokens[0]);
```

Comment:
```suggestion
        conversionResult = convert_from<T>::convert(tokens[0]);
```
Call statically here and in the other spots. Can remove local variables afterwards.